README.md

```diff
@@ -64,6 +65,11 @@ categories:
     - Layout
     - Component
     - Atom
+body_attr:
```

Comment:
Just one question here, would you be able to set attributes as false if desired?

```
body_pops:
    - unresolved
    - some_other_attribute
       - false
  - class:
    - foo
    - bar

Reply:
Also do you want to keep the classes of foo and bar? they don't really hurt anything, just double checking in case this was for demo purposes only

Reply:
For what you've suggested, why not use object instead
`
body_attr:
  unresolved:
  some_other_attribute: false
  class:
    - foo
    - bar
`

That said, it feels like a violation of some unspoken contract to allow the component library to do something to the page. I'm also inclined toward this being implementation documentation rather than pkconfig. Wondering (out loud) if implementors may not have the ability to "share" the body tag this way, and you need an extra abstraction level at least.

Reply:
@SiliconValet re: `violation of some unspoken contract to allow the component library to do something to the page` just so you are aware this is purely for the preview tool and is not meant to import and be used in any other system.  My goal for this was only to allow users to configure the body tag to have classes or attributes that mimic their production environment.  Namely, this lets us add the unresolved tag to the body like in production so that we can test that web components upgrade gracefully.

Reply:
ahhhh, well in that case. okie doke

Reply:
Side note, I added support for the object syntax you suggested.  So you can do:
```
body_attr:
-  unresolved
-  some_other_attribute: false
-  class:
    - foo
    - bar
```

Reply:
https://github.com/PatternBuilder/pattern-kit/pull/32/files#diff-e5ee3e479e6d50ac35a3eeb1dc06b042